libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -108,62 +102,58 @@ class AC_AttitudeControl {
     // Sets and saves the yaw acceleration limit in centidegrees/s/s
     void save_accel_yaw_max(float accel_yaw_max) { _accel_yaw_max = accel_yaw_max; _accel_yaw_max.save(); }
 
-    // Ensure body-frame rate controller has zero errors to relax rate controller output
-    void relax_bf_rate_controller();
+    // Ensure attitude controller have zero errors to relax rate controller output
+    void relax_attitude_controllers();
 
     // Sets yaw target to vehicle heading
-    void set_yaw_target_to_current_heading() { _att_target_euler_rad.z = _ahrs.yaw; }
+    void set_yaw_target_to_current_heading() { _attitude_target_euler_angle.z = _ahrs.yaw; }
 
     // Shifts earth frame yaw target by yaw_shift_cd. yaw_shift_cd should be in centidegrees and is added to the current target heading
     void shift_ef_yaw_target(float yaw_shift_cd);
 
-    // Command an euler roll and pitch angle and an euler yaw rate with angular velocity feedforward and smoothing
-    void input_euler_angle_roll_pitch_euler_rate_yaw_smooth(float euler_roll_angle_cd, float euler_pitch_angle_cd, float euler_yaw_rate_cds, float smoothing_gain);
+    // Command a Quaternion attitude with feedforward and smoothing
+    void input_quaternion(Quaternion attitude_desired_quat, float smoothing_gain);
```

Comment:
Is this function going to be used in a near future? It is unused now and we generally don't merge unused code.